app/views/pages/_editor_guide_text.html.erb

```diff
@@ -113,369 +115,82 @@
-    </p>
-    <ul>
-      <li>The question ID is: <code style="color: aquamarine; background-color: black;">24789130</code></li>
-    </ul>
-    <pre>{% stackoverflow <span style="color: aquamarine;">24789130</span> %}</pre>
-    <p>
-      For other Stack Exchange network sites, you'll need to provide the site's name and use <code>{% stackexchange %}</code> as the tag. For example:
-      <a href="https://diy.stackexchange.com/questions/169988/base-for-refrigerator-wine-shelf">https://diy.stackexchange.com/questions/169988/base-for-refrigerator-wine-shelf</a>
-    </p>
-    <ul>
-      <li>The question ID is: <code style="color: aquamarine; background-color: black;">169988</code></li>
-      <li>The site is <code style="color: orange; background-color: black;">diy</code></li>
-    </ul>
-    <pre>{% stackexchange <span style="color: aquamarine;">169988</span> <span style="color: orange;">diy</span> %}</pre>
-    <p>
-      For answers, you can get the answer's ID code by from the answer's "Share" link. For example:
-      <a href="https://diy.stackexchange.com/a/170185">https://diy.stackexchange.com/a/170185</a>
-    </p>
-    <ul>
-      <li>The answer ID is: <code style="color: aquamarine; background-color: black;">170185</code></li>
-      <li>The site is <code style="color: orange; background-color: black;">diy</code></li>
-    </ul>
-    <pre>{% stackexchange <span style="color: aquamarine;">170185</span> <span style="color: orange;">diy</span> %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "wikipedia-embed", title: "Wikipedia Embed" } %>
-    <p>Enter the full URL of the Wikipedia article you want to embed, with or without the anchor.</p>
-    <pre>{% wikipedia https://en.wikipedia.org/wiki/Wikipedia %}<br>{% wikipedia https://en.wikipedia.org/wiki/Wikipedia#Diversity %}</pre>
-
-    <%= render partial: "editor_guide_h3", locals: { id: "asciinema-embed", title: "Asciinema Embed" } %>
-    <p>All you need is an Asciinema id or URL:</p>
-    <pre>{% asciinema 239367 %}<br>{% asciinema https://asciinema.org/a/239367 %}</pre>
+    <h4>RunKit</h4>
+    <p>Put executable code within a runkit liquid block, as follows:</p>
+    <pre>{% runkit<br>// hidden setup JavaScript code goes in this preamble area<br>const hiddenVar = 42<br>%}<br>// visible, reader-editable JavaScript code goes here<br>console.log(hiddenVar)<br>{% endrunkit %}<br></pre>
 
-    <%= render partial: "editor_guide_h3", locals: { id: "reddit-tag", title: "Reddit Tag" } %>
-    <p>Enter the full URL of the post you want to embed</p>
-    <pre>{% reddit https://www.reddit.com/r/aww/comments/ag3s4b/ive_waited_28_years_to_finally_havr_my_first_pet %}</pre>
+    <% if @user_approved_liquid_tags.include? UserSubscriptionTag %>
+      <h4><%= community_name %> User Subscriptions</h4>
+      <p class="fs-s fw-bold">This liquid tag can only be created with articles by admins.</p>
```

Comment:
Quick, non-blocking suggestion: let's capitalize the "a" in "admins" to keep things consistent across the codebase! Additionally, I believe that "articles" here should actually read "posts." I never remember what we "officially" call them, though. 😄 
```suggestion
      <p class="fs-s fw-bold">This liquid tag can only be created with posts by Admins.</p>
```